src/util/GlobalChecks.cpp

```diff
@@ -13,6 +14,25 @@
 #include <stdexcept>
 #include <thread>
 
+namespace
+{
+void
+printBacktrace()
```

Comment:
maybe this function belongs to `backtrace.cpp/h`

Reply:
sure, I'll make it a single function that prints while unwinding, rather than a pair that allocates a vector and another that consumes it.